README.md

```diff
@@ -94,6 +94,7 @@ For detailed inference benchmarks in more devices and more settings, please refe
 |      ChatGLM2      |    6B     |
 |       Falcon       | 7B - 180B |
 |         YI         | 6B - 34B  |
+|      Mistral       |    7B     |
```

Comment:
Could we update the list of supported models for Gemma？

Reply:
Done